### ISSUE TITLE
eslint-plugin-editorconfig@npm:3.2.0 doesn't provide @typescript-eslint/parser

### ISSUE THREAD

**Issue by xenoterracide** [phanect/eslint-plugin-editorconfig#51]
```
➤ YN0002: │ eslint-plugin-editorconfig@npm:3.2.0 doesn't provide @typescript-eslint/parser (pa98b7), requested by @typescript-eslint/eslint-plugin
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
```
```
❯ yarn explain peer-requirements pa98b7
➤ YN0000: eslint-plugin-editorconfig@npm:3.2.0 doesn't provide @typescript-eslint/parser, breaking the following requirements:

➤ YN0000: @typescript-eslint/eslint-plugin@npm:5.30.0 [76231] → ^5.0.0 ✘
```

```
❯ yarn info @typescript-eslint/parser --name-only
└─ @typescript-eslint/parser@npm:5.30.0
```

```
❯ yarn --version && node --version && uname -a
3.2.1
v16.14.2
Darwin 5c52309d33e3 20.6.0 Darwin Kernel Version 20.6.0: Tue Apr 19 21:04:45 PDT 2022; root:xnu-7195.141.29~1/RELEASE_X86_64 x86_64 i386 Darwin
```

https://yarnpkg.com/advanced/error-codes#yn0002---missing_peer_dependency

workaround locally is to add this to your `.yarnrc`

```yaml
packageExtensions:
  eslint-plugin-editorconfig@3.2.0:
    peerDependencies:
      '@typescript-eslint/parser': '^5'
```

**Comment by phanect** [comment]
Hi @xenoterracide, thanks for reporting this.
I'm looking into this this weekend.

**Comment by phanect** [comment]
@xenoterracide I have released eslint-plugin-editorconfig@4.0.2 and the problem should have been resolved.
Let me know if any problem happens on your end.
